src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -46,28 +46,24 @@ const styles = StyleSheet.create({
     backgroundColor: COLORS.anon_primary
   },
   textContainer: {
-    paddingLeft: BUBBLE_LEFT_PADDING,
+    paddingLeft: BUBBLE_LEFT_PADDING + BUBBLE_LEFT_PADDING,
     paddingRight: BUBBLE_RIGHT_PADDING,
     paddingTop: 4,
     paddingBottom: 4,
-    borderRadius: 8,
-    borderTopEndRadius: 0,
+    borderRadius: 12,
     flex: 1
   },
-  textContainerNewLine: {
-    paddingBottom: 14
-  },
+  textContainerNewLine: {},
   userText: {
     fontFamily: fonts.inter[600],
     fontSize: 12,
-    lineHeight: 19.36,
+    lineHeight: 18,
     color: COLORS.white
   },
   text: {
     fontFamily: fonts.inter[400],
     fontSize: 16,
-    lineHeight: 19.36,
-    marginBottom: 4,
+    lineHeight: 24,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The paddingBottom property from the `textContainerNewLine` style has been removed. If this was intentional and doesn't affect the layout negatively, then it's fine. Otherwise, consider adding it back.
The lineHeight for both `userText` and `text` styles have been changed. Make sure these changes don't affect readability or layout.
```diff
- textContainerNewLine: {
-   paddingBottom: 14
- },
- userText: {
-   lineHeight: 19.36,
- },
- text: {
-   lineHeight: 19.36,
- },
+ textContainerNewLine: {},
+ userText: {
+   lineHeight: 18,
+ },
+ text: {
+   lineHeight: 24,
+ },
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->